Xamarin.Forms.Platform.UAP/WindowsBasePlatformServices.cs

```diff
@@ -145,9 +145,26 @@ void renderingFrameEventHandler(object sender, object args)
 					return;
 				timerTick = newTimerTick;
 				bool result = callback();
-				if (result)
+				if (!result)
```

Comment:
This appears to be modifying the original API `void StartTimer(TimeSpan interval, Func<bool> callback)`, where this PR is to only add a new API. 

Is this code wrong and needs to be updated? 

If I understand the event wire-up code correctly it removes the event handler when the result is true. I believe the intent of the API is to continue if it is true. Something doesn't quite add up here to me.

Given the code snippet below we should stop the timer after the first run since it returns false
```c#
Device.StartTimer(TimeSpan.FromMilliseconds(250), () =>
{
    MyMethod();
    return false;
}
```

After typing this out, I agree with the change. I am confused on how this currently works because the same code I provided above currently works in Xamarin.Forms 4.1 and I don't want to add a change that may cause problems if it reverses the behavior.